external/netstandard/netstandard1.x/project.json.template

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "frameworks": {
+    "{TFM}": {
+      "dependencies": {
+        "NETStandard.Library": "2.0.0-beta-24919-01",
+        "System.Diagnostics.Contracts": "4.3.0"
```

Comment:
This is where we pull in additional NETStandard1.x packages that are needed by our libraries that aren't part of the NETStandard.Library package.  So far all I've needed is this guy, we might need more.